.travis.yml

```diff
@@ -3,7 +3,9 @@ node_js:
   #- "0.10" # EL6, Trusty, Jessie - sadly, jest does not work with 0.10
   - "4.2" # Xenial
   - "4.6" # Fedora 24
-  - "6.9" # EL 7
+  - "4.8" # legacy LTS, Stretch
+  - "6.9" # initial EPEL 7
```

Comment:
I'd like to remove that together with everything non-6.10 when working on raising the minimum requirement as now at least some EL builders don't receive automatic updates and would have 6.9 installed.